contracts/smart-account/modules/OrgUserAdminModule.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/* solhint-disable no-unused-import */
+
+import {BaseAuthorizationModule} from "./BaseAuthorizationModule.sol";
+import {EIP1271_MAGIC_VALUE} from "contracts/smart-account/interfaces/ISignatureValidator.sol";
+import {UserOperation} from "@account-abstraction/contracts/interfaces/UserOperation.sol";
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {IOrgUserAdminModule} from "../interfaces/modules/IOrgUserAdminModule.sol";
+import {IAuthorizationModule} from "../interfaces/IAuthorizationModule.sol";
+import {ISignatureValidator} from "../interfaces/ISignatureValidator.sol";
+
+/**
+ * @author Fil Makarov - <filipp.makarov@biconomy.io>
+ */
+
+contract OrgUserAdminModule {
+    // destination => smartAccount => selector => isAllowed
+    mapping(address => mapping(address => mapping(bytes4 => bool)))
```

Comment:
With recent versions of solidity you can write it directly as 
`mapping(address destination => mapping(address smartAccount => mapping(bytes4 selector => bool isAllowed)))`